tools/locale-nav/locale-nav.js

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable import/no-unresolved */
+import DA_SDK from 'https://da.live/nx/utils/sdk.js';
+import './locale-selector.js';
+
+const ADMIN_SOURCE = 'https://admin.da.live/source/';
+const ADMIN_STATUS = 'https://admin.hlx.page/status/';
+const EDIT_URL = 'https://da.live/edit#/';
+const ROOT_LANG = 'en';
+const ROOT_PATH = '/';
+
+function extractPaths(languages) {
+  const langPaths = languages.data.reduce((acc, { location, locales }) => {
+    if (!location.startsWith('/langstore') && location !== ROOT_PATH) {
+      acc.push(location);
+    }
+    if (locales) acc.push(...locales.split(', ').filter((loc) => loc !== ROOT_PATH));
```

Comment:
Saving `...locales.split(', ').filter((loc) => loc !== ROOT_PATH)` to a variable may help with readability.